templates/email/default/archive.html

```diff
@@ -0,0 +1,58 @@
+[%
```

Comment:
Hmmm... Moving the templates gives me `Template processing error: file error - archive.txt: not found` 😕